### DISCUSSION TITLE
TypeError: Invalid base URL: wss://api.deepgram.com

### DISCUSSION THREAD

**Discussion by athar1711** [deepgram/community#873] [General help]
Hey guys.

I am integrating Deepgram JS SDK in React Native for transcribing a streaming audio in my app. Following this documentation (https://developers.deepgram.com/docs/node-sdk-streaming-transcription)

I initialised the SDK doing this and it initialized successfully:
`import { createClient } from "@deepgram/sdk";`

`const deepgram = createClient("DEEPGRAM_API_KEY");`

But on this line:
`const live = deepgram.listen.live({ model: "nova-2" });`

I am getting this error: **_TypeError: Invalid base URL wss://api.deepgram.com_**

I am working in ReactNative 0.70 and using Deepgram SDK 3.5.0. Can anyone guide me, what is the issue and what I am doing wrong.

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.
